src/DynamicData/List/ObservableListEx.cs

```diff
@@ -298,7 +298,44 @@ public static IObservable<IChangeSet<T>> Bind<T>(this IObservable<IChangeSet<T>>
             throw new ArgumentNullException(nameof(targetCollection));
         }
 
-        var adaptor = new ObservableCollectionAdaptor<T>(targetCollection, resetThreshold);
+        // if user has not specified different defaults, use system wide defaults instead.
+        // This is a hack to retro fit system wide defaults which override the hard coded defaults above
+        var defaults = DynamicDataOptions.BindingOptions;
+
+        var options = resetThreshold == 25
+            ? defaults
+            : defaults with { ResetThreshold = resetThreshold };
+
+        return source.Bind(targetCollection, options);
+    }
+
+    /// <summary>
+    /// Binds a clone of the observable change set to the target observable collection.
+    /// </summary>
+    /// <typeparam name="T">The type of the item.</typeparam>
+    /// <param name="source">The source.</param>
+    /// <param name="targetCollection">The target collection.</param>
+    /// <param name="options">The binding options.</param>
+    /// <returns>An observable which emits the change set.</returns>
+    /// <exception cref="System.ArgumentNullException">
+    /// source
+    /// or
+    /// targetCollection.
+    /// </exception>
+    public static IObservable<IChangeSet<T>> Bind<T>(this IObservable<IChangeSet<T>> source, IObservableCollection<T> targetCollection, BindingOptions options)
+        where T : notnull
+    {
+        if (source is null)
```

Comment:
For consistency with other changes, consider not using the braces here.

Reply:
See comment below